support/@trustfractal/polkadot-utils/src/lib/watcher.ts

```diff
@@ -38,12 +46,12 @@ export class TxnWatcher {
         // unhandled case below.
       } else if (result.status.isInBlock) {
         this.status = 'InBlock';
-        this.onInBlock.callAll();
+        this.onInBlock.callAll( { block: result.status.asInBlock.toHex() } );
```

Comment:
This formatting looks different from other files, did you run a formatter on this?